doc/user/content/guides/postgres-cloud.md

```diff
@@ -123,6 +121,44 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 For more information, see the [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance) documentation.
 
+## Azure Database for PostgreSQL
+
+Before you start, note that a database restart will be required after making changes to the database.
```

Comment:
In the intro:

> Note that enabling logical replication may affect performance.

Since a restart to apply replication configuration is required in all cases, could it be better to add this to the intro instead of specifying it just under the instructions for Azure?

Reply:
For other services, this is added as a step, so maybe also worth adding it in here:

> 1. Restart the database so all changes can take effect.

Reply:
The thing is that the restart happens automatically as soon as you make the change:
<img width="999" alt="image" src="https://user-images.githubusercontent.com/21223421/159279010-b49794d6-c347-4958-b02f-d2936a3f93e8.png">

No separate action is required, so you don't have to click a restart button or anything like that.

This message is just notice so that people are aware that their database will be restarted once they make those changes and can plan for a maintenance window if needed.

Reply:
Same goes for the rest, as soon as you try to make such a config change you are prompted with a save and restart button
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/21223421/159293948-f63ae788-3475-4d88-b436-f66c3acd89f2.png">